benchmarks/CMakeLists.txt

```diff
@@ -8,33 +8,34 @@ macro (do_benchmark source)
 	add_executable (benchmark_${source} ${SOURCES})
 	add_dependencies (benchmark_${source} kdberrors_generated)
 
-	target_link_elektra(benchmark_${source} elektra-kdb elektra-meta)
+	target_link_elektra (benchmark_${source} elektra-kdb elektra-meta)
 
-	set_target_properties (benchmark_${source} PROPERTIES
-			COMPILE_DEFINITIONS HAVE_KDBCONFIG_H)
+	set_target_properties (benchmark_${source} PROPERTIES COMPILE_DEFINITIONS HAVE_KDBCONFIG_H)
 
 	add_custom_target (benchmark_${source}_callgrind
-		rm -f *.out
-		COMMAND valgrind --tool=callgrind --callgrind-out-file=callgrind.out "$<TARGET_FILE:benchmark_${source}>" > output.out
-		# COMMAND strace -o strace.out "${command}"
-		WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
-		)
+			   rm
```

Comment:
strange formatting

Reply:
I added the optional keyword `COMMAND` before the first command, which improved the formatted output considerably.